cmd/kube-scheduler/app/server.go

```diff
@@ -102,7 +102,7 @@ for more information about scheduling and the kube-scheduler component.`,
 	cols, _, _ := term.TerminalSize(cmd.OutOrStdout())
 	cliflag.SetUsageAndHelpFunc(cmd, *namedFlagSets, cols)
 
-	cmd.MarkFlagFilename("config", "yaml", "yml", "json")
+	_ = cmd.MarkFlagFilename("config", "yaml", "yml", "json")
```

Comment:
Why not `klog.Fatalf` the error like above?

Reply:
cmd.MarkFlagFilename is called in many places, but most of them unhandled the error, please reference to: [pull/105342](https://github.com/kubernetes/kubernetes/pull/105342)

do you think we should handled the error ?

Reply:
Modulo version drift, [this](https://github.com/spf13/pflag/blob/2e9d26c8c37aae03e3f9d4e90b7116f5accb7cab/flag.go#L497) is the only place that I can see that MarkFlagFilename can return an error from.

Based on how it's used in the rest of the code base I suggest dropping the `_ =` as that is how it's used in the majority of places.

Reply:
https://github.com/kubernetes/kubernetes/search?q=MarkFlagFilename&type=code

Reply:
thanks for your review!

I agree with you, but I think if we do ignore the err, If we do ignore this error, then we should use `_=` to avoid the lint error.

I think we should refer to https://github.com/kubernetes/kubernetes/blob/1367cca8fd67b09606b01c0a9e46cef59aef3424/cmd/kube-proxy/app/server.go#L523

Reply:
Sorry, I made my point poorly, which was: Pick one form, make it consistent across the code base.

Reply:
yeah, https://github.com/kubernetes/kubernetes/blob/1367cca8fd67b09606b01c0a9e46cef59aef3424/cmd/kube-proxy/app/server.go#L523 is the most recent modification, ` cmd.MarkFlagFilename("config", "yaml", "yml", "json")` is the previous form, I suggest to change it all to `_ = cmd.MarkFlagFilename("config", "yaml", "yml", "json") `

Reply:
Let's log the error at least

Reply:
okay, let's log this error.